CONTRIBUTING.md

```diff
@@ -37,15 +37,39 @@ To get started quickly, we recommend reviewing the following conventions that ar
   * **Incorrect:** *1.2 Configuring Your Che Workspace*
   * **Correct:** *1.2 Configuring your Che workspace*
   
-**Markup conventions**  
+### Markup conventions 
+
```

Comment:
@mishaone, sorry for my late response.  All of these markup conventions are listed in the Red Hat Asciidoc Markup conventions, available to all. Do we want to list them here again?
I understand the case for the first 3 conventions (location then action, gerunds etc).
Imho, after the first 3 conventions we can straight away point to the references. wdyt?

Reply:
Yes, true, @Preeticp. I'm operating under the impression that it's easier (and more likely) for someone not from docs to come in and read a quick review ("here's some quick information about what you need to know" rather than be directed to a separate URL. That's why I've added the links to a separate part. I doubt most people will bother reading it. If we don't agree with this, it's easy enough to just remove. WDYT? Also cc @rkratky so we can have everyone weigh in.

Reply:
I agree with @mishaone that ppl would most likely not be bothered to follow a link. Since Misha already wrote it up, I think we can include it and forget about it. It's not like the RH conventions are going to change significantly any time soon.

Reply:
Cool.

Reply:
FYI, I had a hard time getting the formatting to play nicely for the examples, so I ended up just removing them and linking to the docs conventions directly as Preeti suggested earlier. Fixed in PR4.